plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageMetaData.java

```diff
@@ -114,6 +121,7 @@ protected void loadPreferences(DBPPreferenceStore store)
                 SeparateConnectionBehavior.parse(store.getString(ModelPreferences.META_SEPARATE_CONNECTION))
             ));
             caseSensitiveNamesCheck.setSelection(store.getBoolean(ModelPreferences.META_CASE_SENSITIVE));
+            addExtraDDLInfo.setSelection(store.getBoolean(ModelPreferences.META_EXTRA_DDL_INFO));
```

Comment:
```suggestion
addExtraDDLInfo.setSelection(store.getBoolean(ModelPreferences.META_EXTRA_DDL_INFO));
```

Reply:
I can't understand what is wrong here. Could you please explain it to me?

Reply:
I see it with extra space
![image](https://github.com/dbeaver/dbeaver/assets/28875055/ffcedd62-6470-4b06-8d2b-d681f9d3e1c3)

Reply:
Oh, my bad, there should be space, the previous item is displayed wrong
github view confused me